modules/govuk/manifests/node/s_cache.pp

```diff
@@ -68,10 +68,6 @@
     content => "net.netfilter.nf_conntrack_tcp_timeout_established = 600\n",
   }
 
-  if $router_as_container {
-    include ::govuk_containers::apps::router
-  }
```

Comment:
I think we also need to remove the `$router_as_container` variable

Reply:
Good spot - removed 👍